comprehensive-review/solutions/garden-sensors/src/main/avro/measurement.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "namespace": "com.redhat.training.sensors.model",
+  "type": "record",
+  "name": "SensorMeasurementTaken",
```

Comment:
We are going to agree on the name of this event + field names. This is not a blocker, we can change the fields later